backend/bin/compile_requirements.sh

```diff
@@ -4,10 +4,14 @@ pip-compile --no-emit-index-url --upgrade multi-account/requirements.in
 pip-compile --no-emit-index-url --upgrade multi-account/requirements-dev.in
 pip-compile --no-emit-index-url --upgrade compact-connect/requirements.in
 pip-compile --no-emit-index-url --upgrade compact-connect/requirements-dev.in
+pip-compile --no-emit-index-url --upgrade compact-connect/lambdas/custom-resources/requirements.in
+pip-compile --no-emit-index-url --upgrade compact-connect/lambdas/custom-resources/requirements-dev.in
 pip-compile --no-emit-index-url --upgrade compact-connect/lambdas/license-data/requirements.in
 pip-compile --no-emit-index-url --upgrade compact-connect/lambdas/license-data/requirements-dev.in
```

Comment:
Probably a misunderstanding of the flow on my end, but any clarification appreciated if it's not a bug!

Logged in as Elizabeth Scott, a provider:
<img width="860" alt="image" src="https://github.com/user-attachments/assets/e1210a53-454b-4b7a-ba38-bc1342b73125">


When I click on "Obtain Privileges", nothing happens (I do see a console.log that says the privilege purchase flow starts).

When I manually navigate to `/aslp/Privileges/SelectPrivileges`, I see no options.
<img width="820" alt="image" src="https://github.com/user-attachments/assets/caa97ded-20d9-4998-baf0-8054d15f19f5">

Reply:
@jlkravitz This is where we stubbed off that feature. We can add a comment there if you would like. As of now you need to manually navigate to the select privileges screen. I will look to see if there is something wrong with the compact handling. I would also like to see what data you are getting back from the API there.

Reply:
@landonshumway-ia @ChiefStief 

Response from `/purchases/privileges/options` is

```json
{
    "items": [],
    "pagination": {
        "pageSize": 100,
        "prevLastKey": null,
        "lastKey": null
    }
}
```

The user was created using this command:

```bash
aws cognito-idp admin-create-user --user-pool-id us-east-1_4GGKZNZ9n --username joshua+elizabethscott@magicclay.io --user-attributes Name=custom:compact,Value=aslp Name=custom:providerId,Value=0dd3f8bc-1a40-40a7-95b5-909151106ec5 Name=email,Value=joshua@magicclay.io --temporary-password tempPassword
```

The license data I originally uploaded as a staff user: [licenses.csv](https://github.com/user-attachments/files/17681731/licenses.csv)

DynamoDB Tables:
1. `Sandbox-PersistentStack-CompactConfigurationTable`: 0 items
2. `Sandbox-PersistentStack-LicenseTable`: 0 items
3. `Sandbox-PersistentStack-ProviderTable`: 15 items
4. `Sandbox-PersistentStack-StaffUsersGreenUsersTable`: 1 item

Reply:
In looking into the front end code and the seeing that response body I conclude that there is not a front end bug for this item. We are simply displaying no privileges as none were returned

Reply:
@jlkravitz Ok, the compact/privilege configuration is not being uploaded into the DB. This is likely due to the `activeEnvironments` not including the environment name you are deploying into. Currently this is the value of `activeEnvironments` for `aslp` configuration:
```
activeEnvironments: ["test"]
```

Based on the names of your tables, It looks like you are deploying to a sandbox account. Is that correct, or was this deployed through a pipeline? If it's a pipeline, you would need to add the name of your environment to the list of `activeEnvironments`. 

If it's a sandbox deployment directly from your local machine, then it looks like there is an issue with the compact configuration uploader config custom resource, and we would probably want to create a ticket to look into why your account in particular was unable to deploy the configuration.

Reply:
That's correct. Deployed via CLI using `cdk deploy "Sandbox/*"`.

Where would the compact configuration normally be uploaded from? Do I need to set it properly in JSON?

Reply:
@jlkravitz it should be a completely automated process, the CDK stack defines the process to upload the JSON configuration in this construct https://github.com/csg-org/CompactConnect/blob/610555c5824315421da4217496fca77c7beb6f28/backend/compact-connect/stacks/persistent_stack/compact_configuration_upload.py#L114-L169

If it's a sandbox deployment, it should upload the configuration for all jurisdictions/compacts. If it's a pipeline deployment, it only includes the jurisdictions/compacts which have a matching environment name in their list of `activeEnvironments`.

Reply:
There should be a lambda function in your account with the name prefix of `Sandbox-PersistentStack-CompactConfigurationUpload`. This is the lambda that takes the JSON records and uploads them into the DB. If you are familiar with looking up logs for a lambda function, you could inspect this lambda's logs and determine if it was called during your last deployment and if it was, if there were any errors that occurred.

Reply:
@landonshumway-ia looks like nothing is sent to the configuration uploader:

```json
PersistentStack-CompactConfigurationUpload-23Y7XDd1jh1H with payload 
{
    "RequestType": "Create",
    "ServiceToken": "arn:aws:lambda:us-east-1:010526247786:function:Sandbox-PersistentStack-CompactConfigurationUpload-0vgWhLWpGqCX",
    "ResponseURL": "...",
    "StackId": "arn:aws:cloudformation:us-east-1:010526247786:stack/Sandbox-PersistentStack/b94aaab0-9d44-11ef-988c-0affc2c6c12d",
    "RequestId": "dcf54d63-c7d1-491c-89f5-18ce435dc810",
    "LogicalResourceId": "CompactConfigurationUploadCustomResource",
    "ResourceType": "Custom::CompactConfigurationUpload",
    "ResourceProperties": {
        "ServiceToken": "arn:aws:lambda:us-east-1:010526247786:function:Sandbox-PersistentStack-CompactConfigurationUpload-0vgWhLWpGqCX",
        "compact_configuration": "{\"compacts\": [], \"jurisdictions\": {}}"
    }
}
```

Reply:
@jlkravitz I will attempt to reproduce this issue. Just to confirm, in your `cdk.context.json` file, do you have the following: 
```
"sandbox": true,
```

Reply:
Yup, here is my json file:

```json
{
  "sandbox": true,
  "environment_name": "joshua",
  "ssm_context": {
    "github_repo_string": "csg-org/CompactConnect",
    "app_name": "compact-connect",
    "environments": {
      "joshua": {
        "account_id": "<my account ID>",
        "region": "us-east-1",
        "allow_local_ui": true,
        "security_profile": "VULNERABLE",
        "notifications": {
          "ses_operations_support_email": "<my email>"
          "email": [
            "<my email>"
          ]
        }
      }
    }
  }
}
```